src/test/test-memstream-util.c

```diff
@@ -6,18 +6,18 @@
 TEST(memstream_done) {
         _cleanup_(memstream_done) MemStream m = {};
 
-        assert_se(memstream_init(&m));
+        ASSERT_NOT_NULL(memstream_init(&m));
 }
 
 TEST(memstream_empty) {
         _cleanup_(memstream_done) MemStream m = {};
         _cleanup_free_ char *buf = NULL;
         size_t sz;
 
-        assert_se(memstream_init(&m));
-        assert_se(memstream_finalize(&m, &buf, &sz) >= 0);
+        ASSERT_NOT_NULL(memstream_init(&m));
+        ASSERT_OK(memstream_finalize(&m, &buf, &sz) >= 0);
         ASSERT_STREQ(buf, "");
-        assert_se(sz == 0);
+        ASSERT_OK_ZERO(sz);
```

Comment:
ASSERT_OK and friends should be used for function call.
In this case, please use `ASSERT_EQ(sz, 0)`.